index.js

```diff
@@ -1,14 +1,36 @@
 const express = require('express');
 const cors = require('cors');
-
 const bcrypt = require('bcryptjs');
+const session = require('express-session');
+const protects = require('./middleware.js')
```

Comment:
Since you only have once middleware function in this 'protects' object, you could just [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) it off:
```javascript
const { protected } = require('./middleware');
// ...
```
and then, later on down in your code:
```javascript
server.get("/api/users", protected,  (req, res) => {
  // ...
```